msg/aanderaa_data.msg

```diff
@@ -0,0 +1,14 @@
+float_64 abs_speed_cm_s;
+float_64 direction_deg_m;
+float_64 north_cm_s;
+float_64 east_cm_s;
+float_64 heading_deg_m;
+float_64 tilt_x_deg;
+float_64 tilt_y_deg;
+float_64 sp_std_cm_s;
+float_64 strength_db;
```

Comment:
What are these two?
- `sp_std_cm_s` — something in cm/s... maybe a standard deviation of something? Is sp short for speed?
- `strength_db` — something in decibels? strength of what?

Probably need slightly more verbose names to be clear.

Reply:
Oh interesting — I have more questions, but they're better answered by the aanderaa manual — thank you!